pymc/sampling/mcmc.py

```diff
@@ -563,21 +577,22 @@ def sample(
             draws=draws,
             tune=tune,
             chains=chains,
-            target_accept=kwargs.pop("nuts", {}).get("target_accept", 0.8),
             random_seed=random_seed,
             initvals=initvals,
             model=model,
             progressbar=progressbar,
             idata_kwargs=idata_kwargs,
+            nuts_kwargs=nuts_kwargs,
+            nuts_sampler_kwargs=nuts_sampler_kwargs,
             **kwargs,
+            **nuts_kwargs,
         )
 
     if isinstance(step, list):
         step = CompoundStep(step)
     elif isinstance(step, NUTS) and auto_nuts_init:
-        if "nuts" in kwargs:
-            nuts_kwargs = kwargs.pop("nuts")
-            [kwargs.setdefault(k, v) for k, v in nuts_kwargs.items()]
+        # for k, v in nuts_kwargs.items():
+        #     kwargs.setdefault(k, v)
```

Comment:
Was this causing problems?

Reply:
I thought it was, but does not seem to be still working on it.